KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -612,5 +549,107 @@ function check_phantom_arguments(nd, ss, us)
             ssy = ss[2]
         end
     end
+    
     return ssx, ssy, ssz, usx, usy, usz
 end
+
+"""
+    T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(class, tissue_properties = nothing)
+
+This function returns the default brain tissue properties using a class identifier Matrix
+# Arguments
+- `class` : (`::Matrix`) the class identifier matrix of the phantom
+- `tissue_properties` : (`::Dict`, `=nothing`) phantom tissue properties in ms and Hz considering the available tissues
+
+# Returns
+- `T1, T2, T2s, ρ, Δw`: (`::Matrix`) matrices of the same size of class with the tissues properties information
+
+# Examples
+```julia-repl
+julia> T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(class, tissue_properties)
+
+julia> T1, T2, T2s, ρ, Δw = default_brain_tissue_properties(class)
+```
+"""
+function default_brain_tissue_properties(class, tissue_properties = nothing)
+    if isnothing(tissue_properties)
+        # Load default tissue properties
+        tissue_properties = Dict(
+            # T1, T2, T2*, ρ, Δw
+            "CSF" => [2569, 329, 58, 1, 0],
+            "GM" => [833, 83, 69, 0.86, 0],
+            "WM" => [500, 70, 61, 0.77, 0],
+            "FAT1" => [350, 70, 58, 1, -220],
+            "MUSCLE" => [900, 47, 30, 1, 0],
+            "SKIN/MUSCLE" => [569, 329, 58, 1, 0],
+            "SKULL" => [0, 0, 0, 0, 0],
+            "VESSELS" => [0, 0, 0, 0, 0],
+            "FAT2" => [500, 70, 61, 0.77, -220],
+            "DURA" => [2569, 329, 58, 1, 0],
+            "MARROW" => [500, 70, 61, 0.77, 0])
+        # Define spin property vectors
+    end
```

Comment:
It is better to `merge` the `tissue_properties` (user defined) with `default_tissue_properties` instead of adding an `if isnothing`.